docs/pages/03.06.01-node-customization.md

```diff
@@ -207,18 +229,28 @@ The possible node layout properties are shown below with the values they have wh
     // this: { x_pos: 5, y_pos: 10, pos: "topRight" }. x_pos and y_pos are
     // offsets from the pos point on the node.
     // The order of the elements corresponds to the order of ports in the
-    // outputs array for the node.
+    // outputs array for the node. If there are more input ports than elements
+    // in the array, the last element will be used for all remaining ports.
     outputPortPositions: [
         { x_pos: 0, y_pos: 20, pos: "topRight" }
     ],
 
-    // The 'guide' is the object drawn at the mouse position as a new line
-    // is being dragged outwards.
-    // Object for output port guide can be "circle" or "image".
-    outputPortGuideObject: "circle",
-
-    // If output port guide object is "image" use this image.
-    outputPortGuideImage: "",
+    // An array of elements to control display of output port guide objects.
+    // That is the object drawn at the end of a new link as it is being dragged.
+    // Each element can have a number of different structures like this:
+    // Either
+    // { type: "circle" }
+    // Or
+    // { type: "image", src: "path/picture.svg", width: 10, height: 10 }
+    // Or
+    // { type: "jsx", src: (<FaceCool />), width: 16, height: 16 }
+    //
+    // The order of the elements corresponds to the order of ports in the
+    // inputs array for the node. If there are more input ports than elements
```

Comment:
typo, should be `output`

Reply:
I copied and pasted the whole text from the layout-dimensions.js file so this is fixed